benchmark/ycsb.js

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const bounds = require('binary-search-bounds');
+const dedent = require('dedent');
+const fs = require('fs');
+const path = require('path');
+const stats = require('stats-lite');
+
+const Spanner = require('../src');
+const Workload = require('./workload');
+
+require('yargs')
+  .version(false)
+  .strict()
+  .command(
+    'run [args]',
+    'Run the workload',
```

Comment:
we also want to test with multiple instance of workload running in parallel. So can you add a parameter for that and based on that start n numbers of workload?

Reply:
Done!